Ginger/GingerCoreNET/WorkSpaceLib/ReportsInfo.cs

```diff
@@ -17,17 +17,23 @@
 #endregion
 
 
+using System;
 using System.IO;
 
 namespace Amdocs.Ginger.CoreNET.WorkSpaceLib
 {
     public class ReportsInfo
     {
+        private static string mEmailReportTempFolder = null;
         public string EmailReportTempFolder
         {
             get
             {
-                return Path.Combine(Path.GetDirectoryName(Path.GetTempFileName()), "Ginger_Email_Reports");
+                if (mEmailReportTempFolder == null)
+                {
+                    mEmailReportTempFolder = Path.Combine(Path.GetDirectoryName(Path.GetTempFileName()), string.Format("Ginger_Email_Reports_{0}", DateTime.Now.ToString("dd-MM-yyyy_HH-mm-ss-fff")));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make the enclosing instance property 'static' or remove this set on the 'static' field.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5264586)